app/src/androidTest/java/ch/epfl/sweng/SDP/home/LeaderboardActivityTest.java

```diff
@@ -61,7 +61,7 @@ public void testFriendsButtonsClickable() {
         SystemClock.sleep(3000);
         onView(withTagValue(is((Object)"friendsButton0"))).perform(click());
         onView(withId(R.id.searchField))
-                .perform(typeText("M"), ViewActions.closeSoftKeyboard());
+                .perform(typeText("F"), ViewActions.closeSoftKeyboard());
```

Comment:
Why?

Reply:
Because there is no more Max Muster friend :( . And more generally there is no user that starts with letter M. I think someone removed this user somehow. I am using F for friendsUser